package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Broccoli plugin to precompile inline Handlebars templates",
   "main": "index.js",
   "ember-addon": {
-    "main": "addon.js"
+    "main": "addon.js",
+    "before": ["ember-cli-6to5"]
```

Comment:
I will excise this requirement.

when I was developing it I had renamed the dependency to ember-cli-inline-template-compiler - which by virtue of the alphabet comes after ember-cli-6to5.

In this case, the inline template compiler does NOT like the output of 6to5 with respect to string escaping.  The return template from 6to5 is doubleQuoted and everything within it is then escaped.  I recognize that this may be an upstream failure - the `before` specifier just made it work.

so that it works is currently by virtue of it's naming